browser/ui/common/BUILD.gn

```diff
@@ -0,0 +1,19 @@
+source_set("common") {
```

Comment:
Having another `common` dir under `browser` is not our way.
Instead, you can simply add your file in `browser/ui` directly.
and that file name could be `brave_sheme_utils.h`?

Reply:
Yeah, Sure!

Reply:
Thanks @jagadeshjai I'll run CI for this PR 👍🏼

Reply:
https://github.com/brave/brave-core/pull/24459